eng/Packages.Data.props

```diff
@@ -204,6 +204,13 @@
     <PackageReference Update="Microsoft.AspNetCore.Http.Features" Version="[2.1.1,6.0)" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.AI.Projects'))">
+    <PackageReference Update="OpenAI" Version="[2.8.0,3.0)" />
+    <!-- Foundry libs depend on OpenAI; float SCM with upstream version -->
+    <PackageReference Update="System.ClientModel" Version="*" />
+    <PackageReference Update="System.Text.Json" Version="10.0.0" />
```

Comment:
I'd remove this.   

- The repository is being updated to the 10.x line for BCL packages (and I'd like to wait and merge this after).   

- l also question why we need a direct reference.   We should get this transitively from SCM and MEAI.   Without a direct reference, this should automatically resolve to the higher transitive, unless MEAI doesn't have a direct reference itself.